CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Add `CSR` category to breaking rules.
 - Add support for local bufplugins for `protoc-gen-buf-breaking` and `protoc-gen-buf-lint`.
 - Add RISC-V (64-bit) binaries for Linux to releases.
+- Fix issue with excluding types and empty files when running `buf generate`.
```

Comment:
Nit: Fix type filtering on `buf generate` for empty files, files with no declared types.